reactioncommerce/reaction-admin
# Product table improvements

Impact: minor
Type: style

## Summary
Upgrades to use the latest MUI and related packages. Adds indicator text that shows the number of selected products in the products table.


## Testing
1. Verify there are no console warning
2. Select a product in the products view and verify selection indicator renders correctly.
